sdk/servicebus/service-bus/tsconfig.json

```diff
@@ -3,7 +3,16 @@
     /* Basic Options */
     "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
     "module": "es6" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-    "lib": [] /* lib dependencies are triple-slash directives in lib/index.ts */,
+    "lib": [
```

Comment:
Per the [SDK guidelines](https://azure.github.io/azure-sdk/typescript_design.html#ts-config-lib), we shouldn't populate the `lib` field in tsconfig.json. Instead use the triple-slash reference directives in the files that need these types.